### PR TITLE
chore(docs): upgrading the docs

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/cassandra-pod-delete.md [hunk 1]
```diff
@@ -213,7 +166,7 @@ subjects:
     <td> LIB </td>
     <td> The chaos lib used to inject the chaos </td>
     <td> Optional  </td>
-    <td> Defaults to `litmus`. Supported: `litmus`, `powerfulseal` </td>
+    <td> Defaults to `litmus`. Supported `litmus` only </td>
```
Comment (uditgaurav) [inline docs/cassandra-pod-delete.md]: Inside the tags like `<td>` we can use` <code>litmus</code> `in place of `'litmus'`. This is a common minor mistake we did in HTML tags

Comment (ispeakc0de) [reply docs/cassandra-pod-delete.md]: Is \`litmus\` won't work in td?

Comment (uditgaurav) [reply docs/cassandra-pod-delete.md]: It doesn't support I think.
check this: (https://github.com/litmuschaos/litmus-docs/blob/b09f505935f7af5d42f5f6f628ec2305bb5556b6/docs/cassandra-pod-delete.md#supported-experiment-tunables)

File: docs/openshift-litmus.md [hunk 2]
```diff
@@ -205,7 +166,7 @@ spec:
     appns: 'nginx'
     applabel: 'run=nginx'
     appkind: 'deploymentconfig'
-  chaosServiceAccount: container-kill-sa
+  chaosServiceAccount: delete-sa
```
Comment (uditgaurav) [inline docs/openshift-litmus.md]: `delete-sa`?

File: docs/container-kill.md [hunk 3]
```diff
@@ -36,20 +35,21 @@ sidebar_label: Container Kill
...
     - Containers are killed using the `crictl stop` command.
     - container-kill is run as a pod on the application node. It have ability to kill the application containers multiple times. Which can be varied by `TOTAL_CHAOS_DURATION` and `CHAOS_INTERVAL`.
 - Tests deployment sanity (replica availability & uninterrupted service) and recovery workflow of the application
 - Good for testing recovery of pods having side-car containers
 
 ## Integrations
 
-- Container kill is achieved using the `pumba` or `litmus` chaos library
+- Container kill is achieved using the `litmus` chaos library
+- The container runtime can be choose via setting `CONTAINER_RUNTIME` env. supported values: `docker`, `containerd`, `crio`
```
Comment (uditgaurav) [inline docs/container-kill.md:52]:
```suggestion
- The container runtime can be chosen via setting `CONTAINER_RUNTIME` env. supported values: `docker`, `containerd`, `crio`
```

**Review by uditgaurav — DISMISSED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**